### ISSUE TITLE
Nuxt stable v3.0.0 broken build using beta.20

### ISSUE THREAD

**Issue by samydoesit** [Akryum/floating-vue#912]
Hello Nuxt v3.0.0 hast just released and i have found out that the build is crashing using `"floating-vue": "2.0.0-beta.20",`.

[Stackblitz Repro](https://stackblitz.com/edit/github-ginjft?file=nuxt.config.ts)

## What is happening?
If i try to start the server dev or build it crashes immediately.
```bash
Nuxi 3.0.0                                                                                                18:16:18
Nuxt 3.0.0 with Nitro 1.0.0                                                                               18:16:19
                                                                                                          18:16:19
  > Local:    http://localhost:3000/ 
  > Network:  http://192.168.1.104:3000/


 ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'nuxt')                          18:16:19

  at 𝐝𝐞𝐟𝐚𝐮𝐥𝐭 (node_modules/floating-vue/nuxt.mjs:15:21)
  at Module.installModule (node_modules/@nuxt/kit/dist/index.mjs:487:9)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1842:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6794:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6855:5)   
```

## Expected behavior:
No crash on startup.

**Comment by DamianGlowala** [comment]
Fails right there in my case (`nuxt.mjs`):
![image](https://user-images.githubusercontent.com/48835293/202281969-8a683068-d532-4d15-8abe-963c500a71b1.png)

**Comment by samydoesit** [comment]
Simple workaround for now:
```javascript
// plugins/floating-vue.ts
import { defineNuxtPlugin } from '#app'
import FloatingVue from 'floating-vue'
import 'floating-vue/dist/style.css'

export default defineNuxtPlugin(({ vueApp }) => {
  vueApp.use(FloatingVue)
})
```

**Comment by dpmango** [comment]
@samydoesit It works, thanks

Don't forget to remove  'floating-vue/nuxt' from `modules` array of `nuxt.config.js`

**Comment by BlackSkyJY** [comment]
It works， 
but when i execute the command  `node .output/server/index.mjs` ，Fails right there
![image](https://user-images.githubusercontent.com/29111051/204954386-9a0d9718-f7a5-43ae-81cd-027e63cfe5a7.png)

**Comment by samydoesit** [comment]
@BlackSkyJY In our Nuxt 3 Project `node .output/server/index.mjs` works without problems using the suggested workaround.

Looks like a Problem on your side. Provide a reproduction for further investigation.

**Comment by zc1342303** [comment]
> It works，
> but when i execute the command `node .output/server/index.mjs` ，Fails right there
> ![image](https://user-images.githubusercontent.com/29111051/204954386-9a0d9718-f7a5-43ae-81cd-027e63cfe5a7.png)

Same as me.

**Comment by samydoesit** [comment]
I can only guess maybe a Windows Problem. 🤷‍♂️

Could @zc1342303 and/or @BlackSkyJY open a new Issue because on Unix/Mac platform `node .output/server/index.mjs` is working.

**Comment by cryptopay-pavel** [comment]
I have same issue on my intel macbook.
Nuxt 3.0.0-rc.14 with Nitro 1.0.0
using beta.20

**Comment by abdonrd** [comment]
I also have this error:

> ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'nuxt')     

Using macOS.

**Comment by TheSupremeBeard** [comment]
Any none workaround janky fix for this yet?

**Comment by juliesaia** [comment]
> > It works，
> > but when i execute the command `node .output/server/index.mjs` ，Fails right there
> > ![image](https://user-images.githubusercontent.com/29111051/204954386-9a0d9718-f7a5-43ae-81cd-027e63cfe5a7.png)
> 
> Same as me.

Fixed for me after updating Nuxt to 3.2.3!

**Comment by katerlouis** [comment]
Even with Nuxt 3.2.3, I still get `Cannot read properties of undefined (reading 'nuxt')`
<img width="494" alt="image" src="https://user-images.githubusercontent.com/1983382/222992635-8e65026f-f585-4f77-8f20-f61b0060c95c.png">

Maybe @danielroe can help here?

**Comment by urbgimtam** [comment]
I've successfully installed it in Nuxt 3.4.1 by creating a plugin.

```
// plugins/floating-vue.ts

import FloatingVue from "floating-vue"

export default defineNuxtPlugin((nuxtApp) => {
	nuxtApp.vueApp.use(FloatingVue)
})
```

**Comment by samydoesit** [comment]
Closing, fixed since [v2.0.0-beta.23](https://github.com/Akryum/floating-vue/commit/e064a87507271f93d480e1bd46bea12024204964)
